lib/state/actions/pokemon_list_actions.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:async_redux/async_redux.dart';
+import 'package:pokedex/state/app_state.dart';
+
+class GetPokemonList extends ReduxAction<AppState> {
+  /// TODO: Make a reduce function that will return 20 pokemons to the state
+  @override
+  Future<AppState> reduce() async {
```

Comment:
Do you really need to make this as a Future function for this temporary implementation? This is a bit YAGNI for this case 🙈. Basically you only need to make this `Future` once you're asynchronously waiting for something 🙂 

For further enhancement of your knowledge, kindly research further for the fundamentals of synchronous and asynchronous process 🙂